src/contrib/persistence/Akka.Persistence.Sqlite/ConnectionContext.cs

```diff
@@ -17,7 +17,7 @@ namespace Akka.Persistence.Sqlite
     /// </summary>
     internal static class ConnectionContext
     {
-        private static readonly ConcurrentDictionary<string, SQLiteConnection> Remembered = new ConcurrentDictionary<string, SQLiteConnection>();
+        private static readonly ConcurrentDictionary<string, SqliteConnection> Remembered = new ConcurrentDictionary<string, SqliteConnection>();
```

Comment:
We're breaking binary compatibility here anyway by virtue of switching to a different driver, plus we're trying to get this module out of beta... so I don't have an issue with this.